profiles/coreos/targets/sdk/package.use

```diff
@@ -33,3 +33,6 @@ cross-aarch64-cros-linux-gnu/gdb python_single_target_python3_6 -python_single_t
 
 # Enable gssapi for SDK
 net-dns/bind-tools           gssapi
+
+# Install `perl` with a minimal set of dependencies
+dev-lang/perl minimal
```

Comment:
Is this needed if you already added it to coreos/base profile? Both arm64 and amd64 sdk profiles already inherit from coreos/base profile.

Reply:
ok good to know - so `profiles/sdk` is only to override things from `profiles/base`.